A_Modelica/DroneLibrary/UsersGuide/References.mo

```diff
@@ -2,14 +2,20 @@ within DroneLibrary.UsersGuide;
+<p style=\"margin-left: 30px;\">[Hellerer2014] Hellerer, Matthias, Tobias Bellmann, and Florian Schlegel (2014-03). &ldquo;The DLR Visualization Library - Recent development and applications&rdquo;. In: Proc. 10th Int. Modelica Conf. doi: 10.3384/ecp14096899, pp. 899&ndash;911.</p>
+<p style=\"margin-left: 30px;\">[Kuric2017] Kuric, Muhamed, Nedim Osmic, and Adnan Tahirovic (2017-07). &ldquo;Multirotor Aerial Vehicle modeling in Modelica&rdquo;. In:Proc. 12th Int. Modelica Conf. doi: 10.3384/ecp17132373, pp. 373&ndash;380.</p>
+<p style=\"margin-left: 30px;\">[Luukkonen2011] Luukkonen, Teppo (2011-08). &ldquo;Modelling and Control of Quadcopter&rdquo;. Independent research project in applied mathematics. Espoo, Finland: School of Science, Aalto University.</p>
+<p style=\"margin-left: 30px;\">[Podlaski2020UAV] Podlaski, et al (2020-10). &ldquo;UAV Dynamic System Modeling and Visualization using Modelica and FMI&rdquo;. In: VFS 76th Annu. Forum &amp; Tech. Display. doi: doi : 10 . 4050 / F - 0076 -2020-16289., pp. 1058&ndash;1072.</p>
+<p style=\"margin-left: 30px;\">[Podlaski2021VR] Podlaski, et al (2021). &ldquo;Towards a VR-Based Early Design Interaction for Electric Vertical Take-Off &amp; Landing (eVTOL) Cyber-Physical Models&rdquo;. In: 2021 NAFEMS World Congr. NAFEMS.</p>
+<p style=\"margin-left: 30px;\">[Ma2016] Ma, X., Z. Li, and C. Nae (2016). &ldquo;Multi-domain modeling and Simulation of a Quad-rotor aircraft based on Modelica&rdquo;. In: Proc. Intl. Conf. Modeling, Simulation and Visualization Methods (MSV), pp. 120&ndash;124.</p>
+<p style=\"margin-left: 30px;\">[ModelicaAssociation2014] Modelica Association (2014-07). Functional Mock-up Interface for Model Exchange and Co-Simulation Version 2.0. Tech. rep. Link&ouml;ping: Modelica Association. URL: https://fmi-standard.org.</p>
+<p style=\"margin-left: 30px;\">[Otter2003] Otter, Martin, Hilding Elmqvist, and Sven Erik Mattsson (2003-11). &ldquo;The New Modelica MultiBody Library&rdquo;. In: Proc. 3rd Int. Modelica Conf. Pp. 311&ndash;330.</p>
+<p style=\"margin-left: 30px;\">[Podlaski2021UAV] Podlaski, Meaghan, Robert Niemiec, et al. (2021-05). &ldquo;Multi-Domain Electric Drivetrain Modeling for UAM-Scale eVTOL Aircraft&rdquo;. In: Proceedings of the 77th Annual Forum. The Vertical Flight Society. Virtual.</p>
+<p style=\"margin-left: 30px;\">[Podlaski2022Multicopter] Podlaski, Meaghan, Luigi Vanfretti, et al. (2022). &ldquo;Extending a Multicopter Analysis Tool using Modelica and FMI for Integrated eVTOL Aerodynamic and Electrical Drivetrain Design&rdquo;. In: Proc. American Modelica Conference 2022. doi: 10.3384/ECP2118647, pp. 47&ndash;55.</p>
+<p style=\"margin-left: 30px;\">[Thiele2017] Thiele, Bernhard et al. (2017-05). &ldquo;Towards a Standard-Conform, Platform-Generic and Feature-Rich Modelica Device Drivers Library&rdquo;. In: Proc. 12th International Modelica Conference. doi: 10.3384/ecp17132713, pp. 713&ndash;723.</p>
```

Comment:
Please use `<dl>` lists like the example was showing. Please also sort them alphabetically by the short term. Otherwise, it will be hard to find any of the entries.

Reply:
References have been fixed and put in alphabetical order

Reply:
@mpodlaski You now created a table which is not what the example showed you. 
The example that was given was:
```html
<dl>
<dt>[Example2021]</dt>
<dd><p>John Doe:
        &quot;Amzing paper&quot;,
        Journal artcle, IEEE,
        Trondheim, Norway, 2021.
        (<a href=\"https://hdl.handle.net/11250/2779970\">https://hdl.handle.net/11250/2779970</a>)</p>
</dd>
</dl>
```
Please use that instead of a table.

Reply:
fixed

Reply:
Partially. There were still a lot of syntax errors since you started several lists `<dl>` within each other. I fixed this now in 64d99b3da5b5927bf89d2c0aa3780e1305455c93